Source/AtomicNET/NETNative/NETCInterop.cpp

```diff
@@ -750,7 +751,15 @@ namespace Atomic
         }
 #endif
 
+#if ATOMIC_PROFILING
+        ATOMIC_EXPORT_API void csi_Atomic_Profiler_BeginBlock(Profiler* profiler, const char* name, const char* file, int line, unsigned int argb, unsigned char status)
+        {
```

Comment:
Let's make sure the csi_Atomic_Profiler_BeginBlock is available in the shared library, whether profiling is enabled or not... will avoid missing symbols. Can ifdef the profiler usage, also could add a one time log warning (controlled by a local static bool), which warns if called without profiling support in binary